### ISSUE TITLE
MSI: Cannot build native extensions

### ISSUE THREAD

**Issue by ashie** [fluent-plugins-nursery/td-agent-builder#129]
When I tried to install a C extension of Ruby on Windows, I got the following error:

```
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot find -lx64-msvcrt-ruby270
collect2.exe: error: ld returned 1 exit status
make: *** [Makefile:261: bson_native.so] 繧ｨ繝ｩ繝ｼ 1

make failed, exit code 2

Gem files will remain installed in C:/opt/td-agent/lib/ruby/gems/2.7.0/gems/bson-4.9.4 for inspection.
Results logged to C:/opt/td-agent/lib/ruby/gems/2.7.0/extensions/x64-mingw32/2.7.0/bson-4.9.4/gem_make.out
```

The cause is that td-agent-builder removes libx64-msvcrt-ruby270.dll.a in RubyInstaller.

**Comment by kenhys** [comment]
Need to fix here.
https://github.com/fluent-plugins-nursery/td-agent-builder/blob/master/td-agent/Rakefile#L743

**Comment by ashie** [comment]
A workaround for released [td-agent-4.0.0-x64.msi](https://td-agent-package-browser.herokuapp.com/4/windows):

Extract libx64-msvcrt-ruby270.dll.a in [rubyinstaller-2.7.1-1-x64.7z](https://github.com/oneclick/rubyinstaller2/releases/download/RubyInstaller-2.7.1-1/rubyinstaller-2.7.1-1-x64.7z) and put it into C:\opt\td-agent\lib

**Comment by ashie** [comment]
Fixed via #130
